packages/expo-system-ui/src/SystemUI.ts

```diff
@@ -0,0 +1,295 @@
+import { Platform } from 'expo-modules-core';
+import { ColorValue, processColor } from 'react-native';
+
+import ExpoSystemUI from './ExpoSystemUI';
+
+const assertIsOnPlatform = (functionName: string, onlyAvailableOn: typeof Platform['OS'][]) => {
+  if (!onlyAvailableOn.includes(Platform.OS)) {
+    throw new Error(
+      `"${functionName}" is only available on ${onlyAvailableOn.join(' or ')}! OS: ${Platform.OS}`
+    );
+  }
+};
+
+export type SystemUIBehavior = 'overlay-swipe' | 'inset-swipe' | 'inset-touch';
+
+export type NavigationBarForegroundStyle = 'light' | 'dark';
+export type NavigationBarVisibility = 'visible' | 'hidden';
+export type StatusBarForegroundStyle = 'light' | 'dark';
+export type StatusBarVisibility = 'visible' | 'hidden';
+
+/**
+ * * "light": Light Mode
+ * * "dark": Dark/Night Mode
+ * * "auto": Follow System mode, automatically switch to dark mode.
+ * * "unspecified": Default
+ */
+// export type Appearance = 'light' | 'dark' | 'auto' | 'default';
+
+/**
+ * Changes the Android Status Bar's background color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * SystemUI.setStatusBarBackgroundColor("white");
+ * ```
+ */
+export function setStatusBarBackgroundColor(color: ColorValue): Promise<void> {
+  assertIsOnPlatform('setStatusBarBackgroundColor', ['android']);
+  const colorNumber = processColor(color);
+  return ExpoSystemUI.setStatusBarBackgroundColor(colorNumber);
+}
+/**
+ * Gets the Android Status Bar's background color.
+ *
+ * @platform android
+ *
+ * @example
+ * ```typescript
+ * const color = await SystemUI.getStatusBarBackgroundColor();
+ * ```
+ */
+export function getStatusBarBackgroundColor(): Promise<ColorValue> {
+  assertIsOnPlatform('getStatusBarBackgroundColor', ['android']);
+  return ExpoSystemUI.getStatusBarBackgroundColor();
```

Comment:
wouldn't this just be transparent or somn related on iOS? Maybe null is a better value to use here.